openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcessScriptBuilder.scala

```diff
@@ -633,6 +633,37 @@ class OpenEOProcessScriptBuilder {
     contextStack.head.getOrElse(name, null).asInstanceOf[OpenEOProcess]
   }
 
+  private def arrayContains(arguments:java.util.Map[String,Object]) : OpenEOProcess = {
+    val value = getProcessArg("value")
+    val data = getProcessArg("data")
+
+    val arrayContainsProcess = (context: Map[String, Any]) => (tiles: Seq[Tile]) => {
+      val valueInput: Seq[Tile] = evaluateToTiles(value, context, tiles)
+      val dataInput: Seq[Tile] = evaluateToTiles(data, context, tiles)
+      if (valueInput.size != 1) {
+        throw new IllegalArgumentException(f"The value argument of the array_contains function should resolve to exactly one input, got ${valueInput.size}.")
+      }
+      val theValue = valueInput.head
+      val tile = MultibandTile(dataInput)
+
+      val mutableResult:MutableArrayTile = ArrayTile.empty(BitCellType,tile.cols,tile.rows)
+      for (column <- Range(0, tile.cols)) {
```

Comment:
similar methods use cfor for this triple loop:
https://github.com/Open-EO/openeo-geotrellis-extensions/blob/5233412dafbb499aa49b86a85a1400c961b9ee59/openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcessScriptBuilder.scala#L226

Wondering if we should apply the same pattern, or if this is all equivalent?